tests/flytekit/common/workflows/scala_spark.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from flytekit.sdk.tasks import spark_task, inputs, python_task
+from flytekit.sdk.types import Types
+from flytekit.sdk.workflow import workflow_class, Input
+
+
+scala_spark = spark_task(spark_type="SCALA",
```

Comment:
I think we should do one of 2 things,
Make Spark_type an enum, which it seems it is already in the proto def
OR
we should make special task wrappers like scala_spark etc which fixes the type

Reply:
made an enum .